bigbluebutton-html5/imports/ui/components/error-screen/component.jsx

```diff
@@ -6,15 +6,19 @@ import styles from './styles.scss';
 const intlMessages = defineMessages({
```

Comment:
This is the only place here we need the `defaultMessages` because the client will redirect to an error page if we fail to load the locale from the server.